### ISSUE TITLE
20190415 Github 前端趋势榜

### ISSUE THREAD

**Issue by kujian** [kujian/githubTrending#52]
# 20190415 Github 前端趋势榜
[Github 前端开发日报](http://caibaojian.com/c/news)栏目数据来自[Github热榜](http://news.caibaojian.com/)（我开发的爬虫），每日分享HTML、CSS、JavaScript、Vue等开源项目，为开发者提供动力，点击Star按钮来关注这个项目，点击Watch来收听每日的更新[Github主页](https://github.com/kujian/githubTrending)
## JavaScript
* [ovity/octotree](https://github.com/ovity/octotree) （GitHub code tree on steroids）
* [lib-pku/libpku](https://github.com/lib-pku/libpku) （北京大学课程资料整理）
* [SqueezerIO/squeezer](https://github.com/SqueezerIO/squeezer) （Squeezer Framework - Build serverless dApps）
* [seek-oss/playroom](https://github.com/seek-oss/playroom) （Design with code, powered by your own component library.）
* [Webiny/webiny-js](https://github.com/Webiny/webiny-js) （Serverless CMS (GraphQL + React)）
* [stackshareio/awesome-stacks](https://github.com/stackshareio/awesome-stacks) （A curated list of tech stacks for building different applications &amp; features）
* [LisaDziuba/Awesome-Design-Tools](https://github.com/LisaDziuba/Awesome-Design-Tools) （The best design tools for everything.）
* [twitter/twemoji](https://github.com/twitter/twemoji) （Emoji for everyone. https://twemoji.twitter.com/）
* [vuejs/vue](https://github.com/vuejs/vue) （        A progressive, incrementally-adoptable JavaScript framework for building UI on the web.      ）
* [facebook/react](https://github.com/facebook/react) （A declarative, efficient, and flexible JavaScript library for building user interfaces.）
* [NARKOZ/hacker-scripts](https://github.com/NARKOZ/hacker-scripts) （Based on a true story）
* [ctripcorp/CRN](https://github.com/ctripcorp/CRN) （CRN是Ctrip React Native简称，由携程无线平台研发团队基于React Native框架优化，定制成稳定性和性能更佳、也更适合业务场景的跨平台开发框架。）
* [agalwood/Motrix](https://github.com/agalwood/Motrix) （A full-featured download manger.）
* [ghacksuserjs/ghacks-user.js](https://github.com/ghacksuserjs/ghacks-user.js) （An ongoing comprehensive user.js template for configuring and hardening Firefox privacy, security and anti-fingerprinting）
* [30-seconds/30-seconds-of-code](https://github.com/30-seconds/30-seconds-of-code) （Curated collection of useful Javascript snippets that you can understand in 30 seconds or less.）
* [commenthol/date-holidays](https://github.com/commenthol/date-holidays) （worldwide holidays）
* [alexfoxy/laxxx](https://github.com/alexfoxy/laxxx) （Simple &amp; light weight vanilla javascript plugin to create beautiful animations things when you scrolllll!! Harness the power of the most intuitive interaction and make your websites come alive!）
* [996-icu-avengers/Natasha](https://github.com/996-icu-avengers/Natasha) （旨在各大招聘网站标记996.ICU和955.WLB上榜公司）
* [statsbotco/cube.js](https://github.com/statsbotco/cube.js) （&#x1f4ca; Serverless Analytics Framework）
* [trekhleb/javascript-algorithms](https://github.com/trekhleb/javascript-algorithms) （Algorithms and data structures implemented in JavaScript with explanations and links to further readings）
* [GoogleChrome/puppeteer](https://github.com/GoogleChrome/puppeteer) （Headless Chrome Node API）
* [airbnb/javascript](https://github.com/airbnb/javascript) （        JavaScript Style Guide      ）
* [facebook/create-react-app](https://github.com/facebook/create-react-app) （Create React apps with no build configuration.）
* [mrdoob/three.js](https://github.com/mrdoob/three.js) （        JavaScript 3D library.      ）
* [weilanwl/ColorUI](https://github.com/weilanwl/ColorUI) （鲜亮的高饱和色彩，专注视觉的小程序组件库）
* [Molunerfinn/PicGo](https://github.com/Molunerfinn/PicGo) （A simple &amp; beautiful tool for pictures uploading built by electron-vue）

## Vue
* [PanJiaChen/vue-element-admin](https://github.com/PanJiaChen/vue-element-admin) （        vue2 admin / a management system template      ）
* [dcloudio/uni-app](https://github.com/dcloudio/uni-app) （使用 Vue.js 开发跨平台应用的前端框架）
* [teamprevue/PreVue](https://github.com/teamprevue/PreVue) （&#x1f3a8; Single page application prototyping tool for developers and designers.）
* [ElemeFE/element](https://github.com/ElemeFE/element) （        A Vue.js 2.0 UI Toolkit for Web      ）
* [bestony/logoly](https://github.com/bestony/logoly) （A Pornhub Pattern Logo Generator）
* [fengT-T/996_list](https://github.com/fengT-T/996_list) （996.ICU 和 996.ICU 的匿名投票列表）
* [iview/iview](https://github.com/iview/iview) （        A high quality UI Toolkit with both Vue.js 2.0 and 1.0      ）
* [sendya/ant-design-pro-vue](https://github.com/sendya/ant-design-pro-vue) （??‍???‍? Use Ant Design Vue like a Pro! Demo: https://pro.loacg.com | React: http://pro.ant.design/）
* [vueComponent/ant-design-vue](https://github.com/vueComponent/ant-design-vue) （An enterprise-class UI components based on Ant Design and Vue. ?）
* [macrozheng/mall-admin-web](https://github.com/macrozheng/mall-admin-web) （mall-admin-web是一个电商后台管理系统的前端项目，基于Vue+Element实现。 主要包括商品管理、订单管理、会员管理、促销管理、运营管理、内容管理、统计报表、财务管理、权限管理、设置等功能。）
* [airyland/vux](https://github.com/airyland/vux) （        Mobile web UI Components based on Vue and WeUI (Vue@2.x)      ）
* [Akryum/vue-virtual-scroller](https://github.com/Akryum/vue-virtual-scroller) （Smooth scroll with any amount of data）
* [buefy/buefy](https://github.com/buefy/buefy) （Lightweight UI components for Vue.js based on Bulma）
* [ElemeFE/mint-ui](https://github.com/ElemeFE/mint-ui) （        Mobile UI elements for Vue.js      ）
* [hinesboy/mavonEditor](https://github.com/hinesboy/mavonEditor) （mavonEditor - A markdown editor based on Vue that supports a variety of personalized features）

## HTML
* [FrontendMasters/front-end-handbook-2019](https://github.com/FrontendMasters/front-end-handbook-2019) （[Book] 2019 edition of our front-end development handbook）
* [QSCTech/zju-icicles](https://github.com/QSCTech/zju-icicles) （浙江大学课程攻略共享计划）
* [nndl/nndl.github.io](https://github.com/nndl/nndl.github.io) （《神经网络与深度学习》 Neural Network and Deep Learning）
* [uswds/public-sans](https://github.com/uswds/public-sans) （A strong, neutral, principles-driven, open-source typeface for text or display）
* [jaywcjlove/linux-command](https://github.com/jaywcjlove/linux-command) （        Linux命令大全搜索工具，内容包含Linux命令手册、详解、学习、搜集。      ）
* [kognise/water.css](https://github.com/kognise/water.css) （A just-add-css collection of styles to make simple websites just a little nicer）
* [hvrc/Bunshi](https://github.com/hvrc/Bunshi) （Displays the bond-line structure of a compound or molecule. Gets images from PubChem.）
* [fengdu78/Coursera-ML-AndrewNg-Notes](https://github.com/fengdu78/Coursera-ML-AndrewNg-Notes) （吴恩达老师的机器学习课程个人笔记）
* [javascript-tutorial/en.javascript.info](https://github.com/javascript-tutorial/en.javascript.info) （Modern JavaScript Tutorial）
* [golang101/golang101](https://github.com/golang101/golang101) （Go语言101 : 一本侧重于Go语言语法和语义的编程解释和指导书）
* [ColorlibHQ/AdminLTE](https://github.com/ColorlibHQ/AdminLTE) （AdminLTE - Free Premium Admin control Panel Theme Based On Bootstrap 3.x）
* [facebookresearch/fastText](https://github.com/facebookresearch/fastText) （Library for fast text representation and classification.）
* [google/styleguide](https://github.com/google/styleguide) （        Style guides for Google-originated open-source projects      ）
* [portainer/portainer](https://github.com/portainer/portainer) （        Simple management UI for Docker      ）
* [mckenziearts/shopper](https://github.com/mckenziearts/shopper) （An eCommerce administration built with Laravel 5 for create online shop.）
* [lipis/flag-icon-css](https://github.com/lipis/flag-icon-css) （        ? A collection of all country flags in SVG — plus the CSS for easier integration      ）
* [GetPublii/Publii](https://github.com/GetPublii/Publii) （Publii is a desktop-based CMS for Windows and Mac that makes creating static websites fast and hassle-free, even for beginners.）
* [jimmywarting/StreamSaver.js](https://github.com/jimmywarting/StreamSaver.js) （StreamSaver writes stream to the filesystem directly asynchronous）
* [lib-pku/lib-pku.github.io](https://github.com/lib-pku/lib-pku.github.io) （北京大学课程资料整理）
* [wesbos/JavaScript30](https://github.com/wesbos/JavaScript30) （        30 Day Vanilla JS Challenge      ）
* [uikit/uikit](https://github.com/uikit/uikit) （A lightweight and modular front-end framework for developing fast and powerful web interfaces）

## CSS
* [DivySrivastava/leaflet](https://github.com/DivySrivastava/leaflet) （A social networking platform made using Node.js/ExpressJs/MongoDb(Mongoose)/Angluar/JQuery）
* [getgridea/gridea](https://github.com/getgridea/gridea) （✍️一个静态博客写作客户端 (A static blog writing client)）
* [ipproductions/steveOS](https://github.com/ipproductions/steveOS) （The ultimate mac total transformation pack for Ubuntu GNOME.）
* [jgthms/bulma](https://github.com/jgthms/bulma) （        Modern CSS framework based on Flexbox      ）
* [twbs/bootstrap](https://github.com/twbs/bootstrap) （The most popular HTML, CSS, and JavaScript framework for developing responsive, mobile first projects on the web.      ）
* [tailwindcss/tailwindcss](https://github.com/tailwindcss/tailwindcss) （A utility-first CSS framework for rapid UI development.）
* [necolas/normalize.css](https://github.com/necolas/normalize.css) （A collection of HTML element and attribute style-normalizations      ）
* [zhaoolee/StarsAndClown](https://github.com/zhaoolee/StarsAndClown) （☀️Github星聚弃疗榜,为Github优秀项目写一封推荐信，让Github优秀项目造福人类~ Github apocalypse list, write a letter of recommendation for Github's outstanding projects, and let Github excellent projects benefit humanity~）
* [theme-next/hexo-theme-next](https://github.com/theme-next/hexo-theme-next) （Elegant and powerful theme for Hexo.）
* [Microsoft/TypeScript-Node-Starter](https://github.com/Microsoft/TypeScript-Node-Starter) （        A starter template for TypeScript and Node with a detailed README describing how to use the two together.      ）
* [apachecn/hands-on-ml-zh](https://github.com/apachecn/hands-on-ml-zh) （&#x1f4d6; [译] Sklearn 与 TensorFlow 机器学习实用指南）
* [uswds/uswds](https://github.com/uswds/uswds) （The U.S. Web Design System helps the federal government build fast, accessible, mobile-friendly websites.）
* [rstacruz/cheatsheets](https://github.com/rstacruz/cheatsheets) （My cheatsheets）
* [the1812/Bilibili-Evolved](https://github.com/the1812/Bilibili-Evolved) （增强哔哩哔哩Web端体验: 下载视频, 封面, 弹幕; 自定义播放器的画质, 模式, 布局; 删除广告, 使用夜间模式, 修复界面瑕疵; 以及增加对触屏设备的支持等.）
* [Remix-Design/remixicon](https://github.com/Remix-Design/remixicon) （Open source neutral style icon system）
* [Hackxiaoya/CuteOne](https://github.com/Hackxiaoya/CuteOne) （演示站）
* [google/material-design-icons](https://github.com/google/material-design-icons) （        Material Design icons by Google      ）
* [StylishThemes/GitHub-Dark](https://github.com/StylishThemes/GitHub-Dark) （         Dark GitHub style      ）
* [deeplearning-ai/machine-learning-yearning-cn](https://github.com/deeplearning-ai/machine-learning-yearning-cn) （Machine Learning Yearning 中文版 - 《机器学习训练秘籍》 - Andrew Ng 著）
* [luangjokaj/wordpressify](https://github.com/luangjokaj/wordpressify) （? A build system designed to automate your WordPress development workflow.）
* [elenapan/dotfiles](https://github.com/elenapan/dotfiles) （My custom scripts and configuration files）
* [github/personal-website](https://github.com/github/personal-website) （Code that'll help you kickstart a personal website that showcases your work as a software developer.）
* [hwayne/learntla](https://github.com/hwayne/learntla) （A TLA+ guide）
* [resoai/TileBoard](https://github.com/resoai/TileBoard) （A simple yet highly configurable Dashboard for HomeAssistant）

## TypeScript
* [jhuangtw-dev/xg2xg](https://github.com/jhuangtw-dev/xg2xg) （by ex-googlers, for ex-googlers - a lookup table of similar tech &amp; services）
* [codex-team/editor.js](https://github.com/codex-team/editor.js) （A block-styled editor with clean JSON output）

## Unknow
* [Awesome-Windows/Awesome](https://github.com/Awesome-Windows/Awesome) （        ? An awesome &amp; curated list of best applications and tools for Windows.      ）
* [h5bp/Front-end-Developer-Interview-Questions](https://github.com/h5bp/Front-end-Developer-Interview-Questions) （        A list of helpful front-end related questions you can use to interview potential candidates, test yourself or completely ignore.      ）

## 关于我
 [前端开发博客](http://caibaojian.com/) 站长，[微博](https://weibo.com/kujian) 粉丝18万，[微信公众号](http://t.cn/EV7pcM1)。
如果你觉得我的分享对你有帮助，不妨考虑打赏我[一杯咖啡](http://caibaojian.com/donation)，以资鼓励。
![打赏](https://upload-images.jianshu.io/upload_images/570843-db4053c67a8c9ea9.png)
## 推荐
1. [掘金小册8折优惠，还有全网优惠教程](http://caibaojian.com/goods)
2. [加我微信，进500人微信群，领100本前端电子书](http://t.cn/EV7pcM1)
## 微信群
加入微信群，每天晚上我会汇总一天最好的文章给你。
![qianduanquan](https://user-images.githubusercontent.com/3055447/38468989-651132ac-3b80-11e8-8e6b-15122322a9d7.png)
